pkg/operator/starter.go

```diff
@@ -251,6 +253,11 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	)
 
+	kubeInformers := informers.NewSharedInformerFactory(kubeClient, 10*time.Minute)
+	rbacController := rbaccontroller.NewRBACController(kubeClient,
+		kubeInformers,
+		controllerConfig.EventRecorder)
+
```

Comment:
does kubeInformersForNamespaces work for cluster-wide resources too?

Reply:
Namespace informer will not work for this case. As we are not limiting the informer to a namespace here, shouldn't it work in this case? Please correct me if I am wrong.

Reply:
Line 71 has the empty namespace. My assumption was that this mean cluster-wide resources?

Reply:
Yes, it means cluster-wide resources.

Reply:
Can't we use it instead of another informer factory?

Reply:
Good point. Fixed it to re-use informer factory.